libr/anal/fcn.c

```diff
@@ -958,7 +958,9 @@ static int fcn_recurse(RAnal *anal, RAnalFunction *fcn, ut64 addr, ut8 *buf, ut6
 			return R_ANAL_RET_ERROR;
 		}
 		if ((oplen = r_anal_op (anal, &op, addr + idx, buf + addrbytes * idx, len - addrbytes * idx, R_ANAL_OP_MASK_ALL)) < 1) {
-			eprintf ("Invalid instruction (possibly truncated) at 0x%"PFMT64x"\n", addr + idx);
+			if (!anal->coreb.core || !anal->coreb.cfggeti || !anal->coreb.cfggeti (anal->coreb.core, "cfg.debug")) { // HACK
```

Comment:
see the cfg.debug callback, it's setting this:

                core->io->debug = node->i_value;

as well as this:

                core->bin->is_debugger = true;

Reply:
better check one of those instead of calling cfg.get() all the time